### DISCUSSION TITLE
Project-Specific Fuzzy-Finding

### DISCUSSION THREAD

**Discussion by barrettruth** [ibhagwan/fzf-lua#1514] [General]
Hi,

Was looking for a way to grep/fd over various projects and setting custom args/ignore flags.

Where in my workflow does this make the most sense to add - i.e. in fzf-lua config, in a .nvim.lua, etc...

**Comment by ibhagwan** [comment]
This is a somewhat vague description of your needs/workflow but it sounds to me like a great case for custom pickers/keybinds in your nvim config, you could also create a directory/project switcher which calls fzf-lua’s setup with different args for each project.

**Reply by barrettruth** [reply]
Sorry - just meant like having specific fdignore/rgignore settings per project. sounds like - I should just make those files in my project! lol
